test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -2518,6 +2518,33 @@ var contentTypesReader
             }
         }
 
+        [Fact]
+        public void PackageBuilderWorksWithFilesHavingCurrentDirectoryAsTarget()
```

Comment:
Whether or not this is a recommended case, we should test and assert what happens. Users can come to depend on these sort of formats and, in the future, we should know when the behavior is changing (and potentially choose *not* to change the behavior).